privval/signer_listener_endpoint.go

```diff
@@ -25,6 +36,7 @@ type SignerListenerEndpoint struct {
 
 	timeoutAccept time.Duration
 	pingTimer     *time.Ticker
+	pingInterval  time.Duration
```

Comment:
Do we need this? We're only setting up the timer on construction anyway, so it's not really used for anything.

Reply:
I am going to add reset call in the follow up PR

-       // Reset pingTimer to avoid sending unnecessary pings.
-       sl.pingTimer.Reset(sl.pingInterval)

Had to remove it because of go 1.15 requirement.

Reply:
Ah, got it. 👍